src/Sulu/Bundle/WebsiteBundle/Resources/config/services.xml

```diff
@@ -177,11 +177,28 @@
         </service>
 
         <!-- exception controller -->
-        <service id="sulu_website.exception.controller" class="%sulu_website.exception.controller.class%">
+        <service id="sulu_website.exception_controller"
+                 decorates="twig.controller.exception"
+                 class="Sulu\Bundle\WebsiteBundle\Controller\ExceptionController">
+            <argument type="service" id="sulu_website.exception_controller.inner"/>
+            <argument type="service" id="sulu_core.webspace.request_analyzer"/>
+            <argument type="service" id="sulu_website.resolver.parameter" />
             <argument type="service" id="twig" />
             <argument>%kernel.debug%</argument>
-            <argument type="service" id="sulu_website.resolver.parameter" />
+
+            <tag name="sulu.context" context="website"/>
+        </service>
+
+        <service id="sulu_website.exception.controller" alias="twig.controller.exception">
```

Comment:
Does that work as expected? It's not really avoiding a BC break, since `sulu_website.exception.controller` was not the `twig.controller.exception`.